business-registry-dashboard/app/enums/affiliation-types.ts

```diff
@@ -1,6 +1,7 @@
 export enum AffiliationTypes {
   NAME_REQUEST = 'Name Request',
   AMALGAMATION_APPLICATION = 'Amalgamation Application',
+  CONTINUATION_APPLICATION = 'Continuation Application',
```

Comment:
Stumbled upon this, fixed it on the way. I noticed that Continuation Application weren't showing (there's probably a ticket on Olga's board to fix this here 😁 ) I also noticed we have an empty type in the filters (bug). This was because of this as well.

The code below in the affiliations will fix that as well.

Before:
![image](https://github.com/user-attachments/assets/925dfa47-5e23-4287-a8e4-acfb8989e8c0)

After:
![image](https://github.com/user-attachments/assets/27f12a44-633e-46d0-bfc4-e611febb29d8)